app/views/idv/setup_errors/show.html.erb

```diff
@@ -12,6 +12,9 @@
       heading: t('idv.failure.setup.heading'),
     ) do %>
     <p>
-        <%= t('idv.failure.setup.fail_html', support_code: IdentityConfig.store.lexisnexis_threatmetrix_support_code, contact_number: IdentityConfig.store.idv_contact_phone_number) %>
+      <%= t('idv.failure.setup.fail_date_html', date: I18n.l(Date.parse(@two_weeks), format: I18n.t('time.formats.event_date'))) %>
```

Comment:
I think we can use the symbol for the format
```suggestion
      <%= t('idv.failure.setup.fail_date_html', date: I18n.l(@two_weeks, format: :event_date)) %>
```

Reply:
Also I think we have a time component somewhere that can localize timestamps based on browser timezone?

Reply:
Got rid of the `to_s` and simplified the format, thanks! I'll have to experiment with TimeComponent and see if it will give me just a date.